ui/src/widgets/ui-gauge/types/UIGaugeDial.vue

```diff
@@ -417,6 +421,7 @@ export default {
                 })
         },
         onResize () {
+            this.rerenderCount++
```

Comment:
Can you explain the logic here please? I would expect a `ResizeObserver` and to re-run the rendering logic?

Reply:
There's an already a resize event listener in line [#102](https://github.com/FlowFuse/node-red-dashboard/blob/368c378760b0da2ec4842d1d397d97ab59a13b4f/ui/src/widgets/ui-gauge/types/UIGaugeDial.vue#L102) and just calling the gauge re-rendering and updating method didn't help it was essential to force update the entire gauge dial component on resize. Therefore I used an increment value to update the `key` property of the component to force render